.github/workflows/build-ot3-actions.yml

```diff
@@ -46,6 +46,18 @@ jobs:
           token: ${{ github.token }}
           monorepo: ${{ inputs.monorepo-ref }}
           oe-core: ${{ inputs.oe-core-ref }}
+      - name: Check release build
+        id: check-release
+        if:
+          always()
+        run: |
+          echo "checking ${{steps.build-refs.outputs.monorepo}}"
+          build_type="develop"
+          if $(grep "ot3@" -q <<< ${{steps.build-refs.outputs.monorepo}}); then
```

Comment:
i'm a little dubious about this because it relies on a second-order indirection - we're not parsing git tags, we're parsing the output of a github action that we wrote. What if instead of doing this we add it to the [action](https://github.com/Opentrons/oe-core/blob/main/.github/actions/build-refs/action/index.ts), where we can also add tests?